lib/meta-boxes.php

```diff
@@ -1,12 +1,11 @@
 <?php
-
 /* Get post objects for select field options */
 function get_post_objects( $query_args ) {
   $args = wp_parse_args( $query_args, array(
-      'posts_per_page' => -1,
-    ) );
+    'post_type' => 'post',
```

Comment:
oh no, i updated this function from the boilerplate…
maybe i shouldnt
i'll check